### ISSUE TITLE
Add short section 'What is risk data'

### ISSUE THREAD

**Issue by stufraser1** [GFDRR/rdl-doc#7]
Add summary / images from https://www.gfdrr.org/sites/gfdrr/files/publication/Understanding_Risk-Web_Version-rev_1.8.0.pdf

Plus GFDRR video (@cgiovando  link please)

**Comment by matamadio** [comment]
Link broken :(
Is it this? https://www.gfdrr.org/en/publication/proceedings-2018-understanding-risk-forum

**Comment by stufraser1** [comment]
https://documents.worldbank.org/en/publication/documents-reports/documentdetail/203551468184136582/understanding-risk-in-an-evolving-world

**Comment by matamadio** [comment]
So this one
![immagine](https://user-images.githubusercontent.com/44863827/107508731-bfff0500-6ba1-11eb-80b5-abb51a6486a0.png)

**Comment by stufraser1** [comment]
Yes and perhaps some short text description from the accompanying text

**Comment by matamadio** [comment]
Is this ok?
<img width="427" alt="riskdata_rev" src="https://user-images.githubusercontent.com/44863827/107547692-023e3b80-6bce-11eb-907f-77b251d429cc.PNG">

**Comment by stufraser1** [comment]
I was more thinking the original image and text explanation around it
rather than change the image. I think we need a paragraph to go with it

**Comment by pzwsk** [comment]
Hi, this is a good illustration to re-use but again not sure documentation
is the priority at this stage.

Need to focus on MVP first.

Thanks

Le mer. 10 févr. 2021 à 20:36, Stuart Fraser <notifications@github.com> a
écrit :

**Comment by matamadio** [comment]
Check content to be adapted from https://drive.google.com/file/d/1goymavYo0E_d5V3aCfnWOA4wuFbbLbb8/view?usp=sharing

**Comment by matamadio** [comment]
Added original figure (larger text) on [key concepts](https://gfdrr.github.io/rdl-docs/keyconcepts.html) docs page.
